crates/bevy_app/src/app.rs

```diff
@@ -750,6 +755,42 @@ impl App {
         self
     }
 
+    /// Adds a function that will be called when the runner wants to initialize rendering state
+    ///
+    /// A renderer init function `init_fn` is called only once by the app runner once it has
```

Comment:
IMO this needs an explanation that you can have multiple such functions, and how their relative execution order is determined.

Reply:
I've taken a pass at clarifying this.

Additionally I've now also explicitly asserted that it's possible to assume that the runner is required to block all system updates until after the render_init functions have been called (so, for example, it's clearer that you can assume that events sent while building a plugin will survive, and won't be handled, until the render_init functions have been called)